doc/SDK_cross_compile_example.md

```diff
@@ -108,12 +112,20 @@ If you need to provide additional build flags for your cross compile to function
 See this page for a summary of available gcc flags: https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html.
 
 ### Notes
-These instructions have been tested on both the Raspberry Pi 2 and 3.
+These instructions have been tested on both the Raspberry Pi 2, 3 and 4.
 
 ### Known Issues and Circumventions
 
 If you encounter the error _error adding symbols: DSO missing from command line_ try adding a reference to libdl with  _-cl -ldl_ added to your build script command line.
 
+For Raspberry Pi4 the following workaround is neccessary to properly link the samples with pthread.
+Run the following commands and [recompile the Azure IoT C SDK](#Building-the-SDK).
```

Comment:
nit: maybe provide an explanation for what the command is doing?

Reply:
I would say the issue that is linked gives information on that. I'm trying to keep the text small (or at least not growing it too much).